src/main/java/com/uber/cadence/client/WorkflowStub.java

```diff
@@ -76,6 +76,12 @@ CompletableFuture<Void> signalAsyncWithTimeout(
   CompletableFuture<WorkflowExecution> startAsyncWithTimeout(
       long timeout, TimeUnit unit, Object... args);
 
+  void enqueueStart(Object... args);
```

Comment:
Currently we only have documentation on the main WorkflowClient [methods](https://github.com/uber/cadence-java-client/blob/master/src/main/java/com/uber/cadence/client/WorkflowClient.java#L359) and on the class description for WorkflowClient. I was thinking it would make sense to add it there to be more in line with our current practices.

I'm planning on a followup diff to add in all the overloads to WorkflowClient to try and reduce the scope of this. Adding support to WorkflowStub is just the "minimum" needed to get this working end to end.